src/data/bookstores.json

```diff
@@ -7,7 +7,7 @@
       "address": "603 N Lamar Blvd",
       "city": "Austin",
       "state": "TX",
-      "zip": 78703
+      "zip": "78703"
```

Comment:
Within databases and other data stores, it's generally regarded as good form to save zip codes as strings instead of ints. Otherwise you run into problems with the zip codes you'll see in MA: `01001`, `01003`, etc.